### PR TITLE
Enhance signature of create() for conversion from object

### PR DESCRIPTION
**Pull request by k7sleeper**
Enhanced signature of `create()` to support [Conversion From Object](https://github.com/oozcitak/xmlbuilder-js/wiki/Conversion-From-Object).

### CODE REVIEW

**Comment by k7sleeper** [comment]
@andy-ms: Could you pls review?

**Comment by andy-ms** [comment]
Waiting for review from @wallymathieu

File: types/xmlbuilder/index.d.ts
Comment (wallymathieu) [inline types/xmlbuilder/index.d.ts]: Should the type be Object here or why the restriction?

Comment (k7sleeper) [inline types/xmlbuilder/index.d.ts]: It could, but each value which is not of type string or object with a key/value leads to function `create()` throwing an exception.

Comment (k7sleeper) [inline types/xmlbuilder/index.d.ts]: If we choose `Object` as the type for the first argument then values of type `number`, `array`, `boolean` are also allowed. `create()`called with any of these values throws an exception.

Comment (wallymathieu) [inline types/xmlbuilder/index.d.ts]: Oki 👍

**Review by wallymathieu — APPROVED** [review]

**Pull request merged**
